docs/configuration.md

```diff
@@ -162,3 +146,22 @@ spec:
   featureGates:
     deployTektonTaskResources: true
 ```
+
+### `deployVmConsoleProxy`
+
+Set the `deployVmConsoleProxy` feature gate to true to allow the operator
+to deploy VM console proxy resources.
+
+Resources will be deployed that provide access to the VNC console of a KubeVirt VM,
```

Comment:
So as I just learned, when https://github.com/kubevirt/vm-console-proxy/pull/10 is merged this isn't true anymore. It's the exact opposite then.